roles/openshift_node_upgrade/tasks/main.yml

```diff
@@ -25,14 +28,19 @@
   when: not openshift.common.is_containerized | bool
 
 - name: Remove obsolete docker-sdn-ovs.conf
-  file: path=/etc/systemd/system/docker.service.d/docker-sdn-ovs.conf state=absent
-  when: (deployment_type == 'openshift-enterprise' and openshift_release | version_compare('3.4', '>=')) or (deployment_type == 'origin' and openshift_release | version_compare('1.4', '>='))
+  file:
+    path: "/etc/systemd/system/docker.service.d/docker-sdn-ovs.conf"
+    state: absent
+  when: (deployment_type == 'openshift-enterprise' and openshift_release | version_compare('3.4', '>='))
+     or (deployment_type == 'origin' and openshift_release | version_compare('1.4', '>='))
```

Comment:
Would it make sense to have a filter_plugin for these types of deployment_type/version comparisons?

It's probably not worth it considering that we are going to bring the Origin versioning in line with the OCP versioning in the not to distant future.